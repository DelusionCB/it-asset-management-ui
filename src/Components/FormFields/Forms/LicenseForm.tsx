import React from 'react';
import {Row} from 'reactstrap';

function LicenseForm (): JSX.Element {
    return (
        <>
            <Row>
                <h1>Luo uusi lisenssi</h1>
            </Row>
            <Row>
                <h2>Lisenssin kuvaus</h2>
            </Row>
        </>
    );
}

export default LicenseForm;
