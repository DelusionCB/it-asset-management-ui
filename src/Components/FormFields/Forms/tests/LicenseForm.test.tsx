import React from 'react';
import * as TestRenderer from 'react-test-renderer';
import LicenseForm from '../LicenseForm';
import {BrowserRouter} from 'react-router-dom'
import '@testing-library/jest-dom'

describe('<LicenseForm />', () => {
    let renderer: TestRenderer.ReactTestRenderer;

    // Render the component before tests
    beforeAll(() => {
        renderer = TestRenderer.create(
            <BrowserRouter>
                <LicenseForm />
            </BrowserRouter>,
        );
    });
    describe('Snapshot', () => {
        it('renders correctly', () => {
            expect(renderer.toJSON()).toMatchSnapshot();
        })
    })
})
