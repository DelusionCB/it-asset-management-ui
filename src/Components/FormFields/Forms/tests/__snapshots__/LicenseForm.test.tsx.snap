// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LicenseForm /> Snapshot renders correctly 1`] = `
[
  <div
    className="row"
  >
    <h1>
      Luo uusi lisenssi
    </h1>
  </div>,
  <div
    className="row"
  >
    <h2>
      Lisenssin kuvaus
    </h2>
  </div>,
]
`;
